README.md

```diff
@@ -28,6 +28,22 @@ COMMANDS
   logout       perform a logout
 ```
 
+
+### How to Setup Henesis AutoComplete
+
+```
+$ henesis autocomplete
+```
+
+zsh: 
```

Comment:
I think It would be better explain like "Enter the following script according to your shell type."

Reply:
i agree